django/core/management/commands/makemessages.py

```diff
@@ -208,6 +208,7 @@ class Command(BaseCommand):
     translatable_file_class = TranslatableFile
     build_file_class = BuildFile
 
+    requires_settings = False
```

Comment:
We should probably have a test which loops through all settings with this as False and another with this as True, and asserts based off bad settings (in a subTest)